bin/node/src/main.rs

```diff
@@ -15,11 +15,11 @@ fn main() -> sc_cli::Result<()> {
         .run
         .import_params
         .pruning_params
-        .pruning
-        .replace(String::from("archive"))
-        .map_or(false, |x| x != "archive")
+        .blocks_pruning
```

Comment:
Is there both `blocks_pruning` and `state_pruning` in the config now? 

If so, then this change is not quite equivalent to what was there before  -- I think you just want to disable both types of pruning.

Btw. I think we should have here a little "switch" in the form of a cli argument like `--experimental-pruning` which does not enforce pruning. This way we can test pruning without recompiling over and over again.

Reply:
The "switch" does not belong to this PR though (unless it's trivial).

Reply:
Done as discussed yesterday